akka-docs/src/main/paradox/stream-typed.md

```diff
@@ -1 +1,49 @@
-# Streams
\ No newline at end of file
+# Typed Actors and Streams
+
+@@@ warning
+
+This module is currently marked as @ref:[may change](common/may-change.md) in the sense
+  of being the subject of active research. This means that API or semantics can
+  change without warning or deprecation period and it is not recommended to use
+  this module in production just yet—you have been warned.
+  
+@@@
+
+[Akka Streams](stream/index.md) make it easy to model typesafe message processing pipelines. With typed actors it is possible to extend these pipelines to actors without loosing the type information.
+
+To use the typed stream source and sink factories add the following dependency:
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.typesafe.akka
+  artifact=akka-stream-typed_2.12
+  version=$akka.version$
+}
+
+## Actor Source
+
```

Comment:
mention somehow that these are all "Typed" alternatives to existing integrations, "see also integrating with actors" in the normal docs (stream docs, add a cross ref)